horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/TestService.java

```diff
@@ -228,18 +230,18 @@ List<ExportedLabelValues> listLabelValues(@PathParam("id") int testId,
 
    @GET
    @Path("{id}/export")
-   @APIResponseSchema(value = String.class,
-           responseDescription = "A Test defintion formatted as json",
+   @APIResponseSchema(value = TestExport.class,
+           responseDescription = "A Test definition formatted as json",
            responseCode = "200")
-   String export(@PathParam("id") int testId);
+   TestExport export(@PathParam("id") int testId);
 
    @POST
    @Path("import")
-   @APIResponse(responseCode = "204", description = "Import a new test")
+   @APIResponse(responseCode = "204", description = "Import a new Test or update an existing Test")
    @RequestBody(content = @Content( mediaType = MediaType.APPLICATION_JSON,
-           schema = @Schema( type = SchemaType.STRING, implementation = String.class)) )
-   @Operation(description="Import a previously exported Test")
-   void importTest( String testConfig);
+           schema = @Schema( type = SchemaType.STRING, implementation = TestExport.class)) )
```

Comment:
The OpenApi viewer is displaying the input payload as "string". This should be a JSON representation of `TestExport`

![image](https://github.com/Hyperfoil/Horreum/assets/959822/e8f3bd2a-de77-46f9-95c8-28194cc13607)